frontend-react/src/pages/resources/legacy-page-content/CsvSchemaDocumentation.tsx

```diff
@@ -270,10 +270,11 @@ export const CsvSchemaDocumentation = () => {
                                         {section.title}
                                     </h3>
 
-                                    {section.items?.map((item) => {
+                                    {section.items?.map((item, index) => {
                                         return (
                                             <CsvSchemaDocumentationItem
                                                 item={item}
+                                                key={`CsvSchemaDocumentationItem-${index}`}
```

Comment:
Noticed this key-missing error in the console while debugging, figured I'd fix it.

Reply:
Oh! Good to know, I'll remove this from one of my branches -- I'd plugged in a fix earlier today.

Reply:
We could just make it a different PR and just merge it quickly? It's simple/save.

Reply:
Throw me on for review :)